src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -138,6 +138,26 @@ public void clickShowStats() {
 
     public void clickIndicateMissingResponses() {
         click(indicateMissingResponsesCheckbox);
+        expandPanels();
+    }
+
+    public void expandPanels() {
+        try {
+            // In case the panels are already expanded
```

Comment:
The function expandPanels() here is trying to expand panels for testing to aim for minimal changes in html in this PR. Some actions cause panels to reload.(For example, when resultPage.displayByGiverRecipientQuestion() is clicked, the page would sort the result panels.)  After reloading, the panels would auto expand in the old implemention.

This PR does not auto expand the panels, thus a lot of clicking expand button is required to be added to the test. To not add too many clicking expand buttons, I include this expandPanels() into every other action that might cause a reload of panels. Checking for whether the panels are already expanded is for cases where<br>
&emsp;&emsp;displayByGiverRecipientQuestion() might be called twice in a row and panels are already expanded before second displayByGiverRecipientQuestion() call. When expandPanels() is called second time in second displayByGiverRecipientQuestion() call, the panels are actually collapsed and the test will fail.

There could be two solutions if we want to change it here:
1. Add in a <b>needToExpandPanels</b> param to each action function in InstructorFeedbackResultsPage (e.g. in displayByGiverRecipientQuestion) to indicate whether we want to expand panels following this action.
2. Delete expandPanels function and call clickCollapseExpand() in the test to expand panels for all cases where panels are reloaded and need to be expanded.